torch/nn/quantized/modules/conv.py

```diff
@@ -498,3 +502,64 @@ def from_float(cls, mod):
         qconv.zero_point = int(act_zp)
 
         return qconv
+
+
+# === Transposed Convolutions ===
+
+class _ConvTransposeNd(_ConvNd):
+    def __init__(self, in_channels, out_channels, kernel_size, stride,
+                 padding, dilation, transposed, output_padding,
+                 groups, bias, padding_mode):
+        if padding_mode != 'zeros':
+            raise ValueError('Only "zeros" padding mode is supported for {}'.format(self.__class__.__name__))
+
+        super(_ConvTransposeNd, self).__init__(
+            in_channels, out_channels, kernel_size, stride,
+            padding, dilation, transposed, output_padding,
+            groups, bias, padding_mode)
+
+    def _input_padding(self, kernel_size, dilation, padding):
+        # type: (List[int], List[int], List[int]) -> List[int]
+        res = torch.jit.annotate(List[int], [])
+        for kdx in range(len(kernel_size)):
+            pad = (dilation[kdx] * (kernel_size[kdx] - 1) - padding[kdx])
+            res.append(pad)
+        return res
+
+    def _output_padding(self, input, output_size, stride, padding, kernel_size):
```

Comment:
is this the same as: https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/conv.py#L496, can we reuse this code?

Reply:
Yes, I can inherit from the non-quantized class.